### PR TITLE
Feature/user profile

### PR DESCRIPTION
**Pull request by deathmarcus**
UserProfile, user info is obtained dynamically from the db, and rendered in the profile page (in progress)left section is finished, cover photo displays correctly, working on tabs and main section"
useFetch hook added (only works with user info)
ProfileCard is finished, (left part of the page, can be used for other types of profiles or sections)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ import BasicRegistrationPage from "./pages/BasicRegistration/BasicRegistrationPa
 import LoginPage from "./pages/LoginPage/LoginPage";
 import { createTheme, ThemeProvider, styled } from "@mui/material/styles";
 import { green, purple, yellow, grey } from "@mui/material/colors";
+import PhotographerCard from "./components/Cards/PhotographerCard";
```
Comment (luised2301) [inline src/App.js:11]: this line is not in use

File: src/components/PhotographerProfile/ProfileName/ProfileName.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.profile-user-text {
+  display: flex;
+  color: #000000;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 26px;
+  line-height: 31px;
+  align-items: center;
+  text-align: center;
+  margin-top: 25px;
```
Comment (luised2301) [inline src/components/PhotographerProfile/ProfileName/ProfileName.css:10]: try to use the material ui SX={} propertys

File: src/index.css [hunk 3]
```diff
@@ -1,14 +1,15 @@
+html {
+  background-color: #f6f6f6;
+}
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (luised2301) [inline src/index.css:3]: fonts can be provided by the theme on APP

**Review by luised2301 — APPROVED** [review]

File: src/contexts/UserContext.js [hunk 4]
```diff
@@ -1,34 +1,36 @@
...
 import React, { useState, createContext, useEffect, useContext } from "react";
 export const UserContext = createContext();
 
 const UserContextProvider = ({ children }) => {
   const [user, setUser] = useState();
   const [token, setToken] = useState();
 
+  useEffect(() => {
+    console.log("user", user);
+    console.log("token", token);
```
Comment (alepaez-dev) [inline src/contexts/UserContext.js:10]: creo que es innecesario

**Review by alepaez-dev — APPROVED** [review]
@deathmarcus te lo apruebo nomas porque me dijiste que conocias a la morra de belanova, 100 te voy a poner, es mas yo te hago el proyecto.

ta buena la background music de la junta

File: src/contexts/UserContext.js (hunk 4 above)
Comment (deathmarcus) [reply src/contexts/UserContext.js:10]: Si, ya lo voy a eliminar

**Pull request merged**